azurerm/internal/services/storage/resource_arm_storage_account.go

```diff
@@ -95,6 +95,8 @@ func resourceArmStorageAccount() *schema.Resource {
 					"ZRS",
 					"GRS",
 					"RAGRS",
+					"GZRS",
+					"RAGZRS",
```

Comment:
can we add a test-case for these? these can require a newer version of the Azure API